Blockdiagram

dvc94ch opened the pull request:
@KilnOfTheSecondFlame de letschdi commit stellt secher das mer kei trailing whitespace meh hend. Setsch e dim editor chönne istelle das wenn en datei spicherisch das 'abc   \n' zo dem 'abc\n' werd.

@JanOdermatt: Blockdiagram review - Esch meh oder weniger das wo mer gester besproche hend. Mosch säge falls öppis onklar esch.

dvc94ch committed:
Add and adjust Platform documentation.

dvc94ch committed:
Add Programming-Language evaluation for EE board.

dvc94ch committed:
Fix table and remove redundant section.

dvc94ch committed:
Add Electronics Blockdiagram

dvc94ch committed:
Fix trailing whitespace.

dvc94ch committed:
Add trailing whitespace lint for *.tex files.

dvc94ch committed:
Add TODO items.

dvc94ch commented on Projektdokumentation.tex:
```diff
@@ -816,7 +816,18 @@ \section{Morphologischer Kasten Mechanische Komponenten}
+\subsection{Blockdiagram}
+\begin{figure}[h!]
+  \includegraphics[keepaspectratio,width=\textwidth]{BlockdiagramElektronik}
+  \label{fig:ElektronikBlockdiagram}
```
@JanOdermatt Beispiel Kommentar: David was hesch du do veren mist bauet?

JanOdermatt commented:
@dvc94ch 
## Änderungen Blockdiagramm

Frage: Was ist FTDI Breakout im Blockdiagramm? 

Kannst du bitte am Blockdiagramm folgende Anpassungen durchführen:

- [x] Sensoren mit Funktion (z.B. Beschläunigungssensor) beschriften
- [x] Servo 1/2 mit Antrieb bzw. Kran oder Hub beschriften
- [x] Linie ziehen, dete wo zwei Komponente kommunizierid oder strom übertreid wird

Commit:
Diverse Verbesserungen

Verbesserung der technischen Schreibweise und Rechtschreibung in den
Abschnitten 5.1, 5.2

dvc94ch commented:
Das FTDI Breakout ist um den Flash Speicher der sich auf dem Lofive Board befindet zu beschreiben und Software zu debuggen. Es kommt nicht in den Roboter hinein.

dvc94ch commented:
Habe die Änderungen durchgeführt. Stromübertragung habe ich weggelassen da alle Blöcke Strom brauchen...
